rspec-its.gemspec

```diff
@@ -13,6 +13,14 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/rspec/rspec-its"
   spec.license       = "MIT"
 
+  spec.metadata = {
+    'bug_tracker_uri'   => 'https://github.com/rspec/rspec-its/issues',
+    'changelog_uri'     => "https://github.com/rspec/rspec-its/blob/v#{spec.version}/Changelog.md",
+    'documentation_uri' => 'https://rspec.info/documentation/',
```

Comment:
This should link to the rdoc for this gem, its not included in the main gem documentation.

Reply:
Cool. I don't see this on rspec.info. How about: `https://www.rubydoc.info/gems/rspec-its/1.2.0`?

Reply:
Yes thats what I meant :)